assets/js/data/cart/test/push-changes.ts

```diff
@@ -46,8 +46,8 @@ jest.mock( '@wordpress/data', () => ( {
 } ) );
 
 // Mocking lodash here so we can just call the debounced function directly without waiting for debounce.
```

Comment:
```suggestion
// Mocking the debounce method so we can use the callback directly without waiting for debounce.
```

Reply:
Code seems fine, tests pass, functionality is preserved, no bad output on the console.. Apart from this forgotten comment, I haven't got anything to add, stellar work @mikejolley 
Green light for approval 👍🏼